core/src/input/input.tsx

```diff
@@ -28,22 +28,53 @@ const getClass = (props: InputProps): string => {
 };
 
 export interface InputProps {
+	/**
+	 * Type of input. For example: button, email, password, number, etc...
+	 */
```

Comment:
Avoid mentioning "button" here, as it would render a button, which is the case we don't want to support yet (button via input tag).

In fact, we may even block its usage from the code